### DISCUSSION TITLE
Quarg Revamp Discussion

### DISCUSSION THREAD

**Discussion by Saugia** [endless-sky/endless-sky#9016] [General]
We are now continuing the Quarg Revamp project, and I held a stream where we discussed most things overall and what choices we've made so far. [You can find a recording of that stream here.](https://www.youtube.com/watch?v=HJQO9wDVrLo) Below I will summarize what we've come to for a conclusion:

## Design Choices
Will update this with images once we get the design in a more completed state.
- Skylark-esque - mostly dark greys, silvers, and perhaps small cases of glowing lights and light hues of blue-purple.
- Bridge gap between Becca's and Todd's newest designs- not too spiked, not too curved. See the VOD linked above.
- Avoid rocket ship shapes and uniform shape distribution.
- Each Quarg ship class can have small differences in appearance based on their ringworld location - not necessarily stat-focused, but slightly altered sprites, like Archons.

*Update:* The following design is an almost-complete sprite of the direction of the new designs, and I have delegated @1010todd as the modeler to create them:
![test5](https://github.com/endless-sky/endless-sky/assets/93169396/8b2857c9-47f2-41b5-80db-e7b33c3b0756)
![test5_thumb](https://github.com/endless-sky/endless-sky/assets/93169396/30bec8ad-edf7-4dcf-89ce-eff6fca4c9a7)
Note, this is the 2x size. This will ideally be one of the two Heavy Warships listed below, and we'd work smaller from here.

## Outfit Choices

In addition to what is currently in the game, we will likely be pulling from the original [Quarg Expansion PR](https://github.com/endless-sky/endless-sky/pull/3665):
- Fluxion Shield Generator
- Infimum Shield Generator
- Singularity Core
- Tachyon Core
- Central Repair Unit
- Peripheral Repair Unit
- Different-sized Graviton thrusters

## Weapon Choices

No ballistics.

In addition to what is currently in the game:
- Voidscythe - A large, single-blast type beam weapon, with possible niche in damage type(s). Think like the "Nullifier" of turrets.
- Possible others being discussed.

## Ship Choices

No Fighters. No Transports. No Superheavies. Scale generally around Human ship level, depending on design and class. Names based on dragons/dragon mythology.

We've come up with a total of 10-11 ships for the Quarg's roster:
### Civilian Design
- Lindwyrm Freighter (Skylark Replacement)
- Amphithere Freighter
- Kirin Freighter? (A Freighter designed for freight in times of conflict)
### Utility Design
- Guivre Utility Ship (Ringworld construction ship - transports large loads of materials for them and assists in construction for them)
### Legacy Design (Older ships pre-Pug and Coalition Conflict)
- Wardragon Light Warship
- Drake Light Warship
- Direwyrm Medium Warship
### Recent Designs (Due to Pug and Coalition Conflict)
- Wyvern Medium Warship
- Hydra Medium Warship
- Unnamed Heavy Warship (Appears as a rare spawn for the Korath Efret Quarg and during the Coalition Campaign)
- Archdragon Heavy Warship (Only appears within the Coalition Campaign)

## Balance Notes
- A Wardragon (LW) should not be able to take on a Heliarch Punisher. Two Wardragons have a better chance to take on a Punisher, but not easily.
- A Quarg Medium Warship should beat 2 punishers relatively consistently; 3 Punishers should be likely to beat one.
- A Quarg Heavy Warship should require at least 4-5 Punishers to defeat.
- The Skylance needs to be nerfed. The original proposal is in half.
- The Quarg should make use of active cooling more in their outfits (see the old PR).
- The Antimatter Core needs to be rebalanced. (In turn, balance the other cores.)

## Suggestions
I'm leaving these unchecked until we have a general consensus on them being things we do/or we just decide on them anyway.
 - [ ] Perhaps Quarg thrusters can be given reverse thrust equal to their normal thrust.
 - [ ] Perhaps the Quarg should have a lower disable threshold.
 - [ ] Perhaps the Quarg should have lower armor, but more resistances.

Some ship and outfit names are obviously to be decided and are not final, others may be adjusted.

While design and overall ships and outfits/weapons have been decided upon, please use this opportunity to discuss balance as that's the category we need the most feedback in. Again, reference the older PR found [here](https://github.com/endless-sky/endless-sky/pull/3665) as a start. Thank you all for being so patient while we worked up to this point, and I can't wait to see what designs we look at for finals, as well as where the Quarg turn up in the relative future!

_Side-Note: Capn' Pester, the Author ship that uses the current Wardragon design, will keep that design. The name "Skylark" will no longer be used, however it will cause conflicts with pre-existing Quarg ships in saves if it is used elsewhere._

**Comment by Quantumshark** [comment]
My thoughts and suggestions on outfit balance:

## Shield Generators
From the current stats, the Quantum shield generator could do with a moderate buff (at most +50%) - the old Quarg PR covers this. I'd also give it lower energy draw too, and maybe also disruption and piercing resistance?

## Hull Repair
The old Quarg PR covers moving the hull repair from an integrated hull ability to an outfit, which I think is good. 
The hull repair from the PR is balanced for T3 as-is, but I think it might fit better thematically for the Quarg to mostly rely on shields over hull and use disruption resistance and/or protection, so the hull repair could be less strong than the Heliarch hull repair without making the Quarg vulnerable to disruptor spam.

## Energy
The Antimatter Core needs a _lot_ of nerfing, and their other outfits probably need their energy draws reined in quite a bit to compensate. Even after the major nerf it recieved in the old Quarg PR, it's still vastly too strong. After watching the parts of the stream I missed, I noticed one idea that was mentioned is giving the Quarg very heat-inefficient generators to make them harder to work with when looted. I'm not sure that fits the quarg thematically, but I like it from the balance side. I have worked out two proposals for where they should be - one based on Derpy's idea of giving them extremely high heat per ton, and one going for a much more efficient style.
![Quarg stat proposals](https://github.com/endless-sky/endless-sky/assets/18634983/575604ce-846c-4d19-acc9-d237bf5f9106)

That'd still be enough to easily eclipse every other reactor in the game - The Singularity core would be stronger than a blue sun by more than twice the difference between the blue sun and the strongest human reactors (as well as being smaller), and how often do you run a fusion reactor once you have access to Korath plasma cores or Wanderer suns? - but it wouldn't *completely* shatter the concept of energy balance.
And yes, the energy draw of their other outfits is probably going to need to be reduced to compensate.

Their batteries are in a reasonable state balance-wise, but they aren't interesting. That's not necessarily a problem, but it would be nice if they were. If we decide to go with the idea of giving them lots of active cooling, but not with the idea to spread it out across all their outfits, I think this is probably the strongest candidate for where to put it.

## Engines
Engines are in a reasonable state already apart from the aformentioned energy issue, though they don't exactly scream "T3". The suggestion to give them reverse thrust equal to their normal thrust seems reasonable from the balance side, though integrated reverse thrust is already a feature of Remnant engines and is planned for the Xapleaux, so perhaps it'd be better to give Quarg engines something else instead.

## Weapons
The Quarg AM could maybe use a small buff - maybe up the AM strength to 50, to give it more of an advantage over lower-tier AM systems.
Skylances need their DPS cut considerably, and honestly I'd say they should either lose the ion or commit to it.

## Other
The old Quarg PR gave all Quarg outfits integrated active cooling - 15 per ton, at a cost of 1 energy per 5 cooling.
This is nice, and helps make the Quarg less vulnerable to overheating. The energy efficiency of it should probably be improved though - this can be considered to be part of the "the energy draw of their other outfits is probably going to need to be reduced to compensate." point.

I'd like to see their ramscoop moved from their hulls to an outfit, but that's only a minor thing, and need not be done at the same time as other changes.

**Reply by mathwhiz1212** [reply]
I 100% agree with increasing Quarg shield gen since it's currently inferior to Heliarch tech. 

Thematically, having the Quarg cores simply be less powerful rather than heat inefficient seems to fit the Quarg story better.

Some way of cooling as you propose would prevent tier 1-2 ships from taking down Wardragons too easily with heat weapons.

**Reply by Zitchas** [reply]
Merely giving the Quarg engines 100% reverse thrust seems rather "blah" to me.
a) There's already an engine set that provides strong reverse thrust.
b) It's not particularly novel, impressive, or even noteworthy.

I'd like to suggest that we find some more interesting way of buffing the engines; maybe even "solving" the "AM reactor is too good" problem too.

Some ideas:
- Have the thrusters/steering *give* energy when used instead of consuming energy. The ship has to keep moving in order to get power. It's both powerful, but imposes an interesting limitation that could sometimes be problematic to deal with. They would of course still produce heat. Probably even more, since it's providing both thrust AND power.
- Given that they are graviton based engines, how about having them produce significant amounts of "jamming" versus proximity triggered weaponry. Maybe so long as their engines are in-use, they change the trigger radius for proximity triggers. It'd be a pretty powerful buff if - while their engines are in use - proximity based weapons exploded at 25% greater distance than normal. It'd make it harder to dodge, but they'd generally only get hit by a small percentage of the damage (if any), as some explosive weapons have comparable trigger-to-explosion radiuses.

**Reply by Arachi-Lover** [reply]
Provided the extra comes on top of them having the full reverse thrust.

**Reply by Zitchas** [reply]
1) There is literally no use for Quarg ships to have reverse thrust, currently. They are turret based, so they can kite stuff while flying forward just as easily.
2) Having thrusters that *produce* energy and/or trigger proximity explosions at greater than normal range are both far more powerful benefits than any amount of reverse thrust ever could be.

**Reply by Azure3141** [reply]
Since we want to lean into the idea of Quarg technology being truly advanced instead of "engine but marginally better thrust/weight ratio," I think the idea of omnidirectional thrust (reverse as well as lateral, if that ever happens) fits well with a gravity drive based propulsion system. It makes thematic sense for such a drive system to not require the ship to be pointed in a particular direction, and in fact it would be a little weird if behaved exactly like existing engines. In fact, if it was possible I'd say the Quarg drive flare should be more of a warped bubble surrounding the ship instead of merely being glowy purple exhaust flares (what is being exhausted, exactly?).

If that isn't already unique enough, gravity themed propulsion also fits with the `inertia reduction` currently used on Quarg engines. I think it would be interesting if this was turned up significantly, while Quarg reactors were also greatly increased mass (but not outfit space, aside from the usual balancing). This would make Quarg reactors much more difficult to use on other ships without similarly inertialess engines, something much more unique.

I question whether there's really an issue with the Remnant also having reverse thrust, given the amount of reverse thrusters being added onto every species. Unlike a conventional thruster expelling propellant out in a particular direction, you wouldn't need to point the entire ship to have a gravity drive accelerate you off in an arbitrary direction, so if anything it makes less sense for the Remnant to have all that reverse thrust instead of the Quarg. I don't necessarily think that's an issue, but if there's really a concern about it not being appropriate for one or the other, it seems pretty clear which engine set would integrate better with the concept.

**Reply by Zitchas** [reply]
I disagree about the Reverse thrust, (although I agree that it is certainly a lot less special, interesting, or unique than it was back when the Remnant got that change) but your mention of the inertia reduction is a good one. Doubling down on that and seeing what other ways we could build off inertial reduction to make an actual unique engine is good one. Of course, inertial reduction engines from them getting put onto other ships is going to cause havoc for balancing...

**Reply by Azure3141** [reply]
Perhaps it could be paired with high energy use so that they're impractical to run without Quarg reactors.

Or alternatively, perhaps they do something to drag (either via negative `drag reduction` or by just adding to the attribute) to limit maximum speeds, and also mimic inertialess thrust by having ships come to a stop after you stop thrusting.

**Reply by Saugia** [reply]
In general, yes, idea is to have Quarg outfits take up power or some form of energy only Quarg outfits can supply, meaning that if the player were to steal outfits, they'd have to adjust their builds to make up for the more hungry stronger weapons as opposed to little drawback (for example, nabbing a Skylance or two).

**Reply by Arachi-Lover** [reply]
> There is literally no use for Quarg ships to have reverse thrust, currently. They are turret based, so they can kite stuff while flying forward just as easily.

Except being turret based is still a point in favor of reverse thrusting, as it allows for quicker backtracking without needing to turn around, thus making it easier to get back in range of turrets if one overshoots.

> Having thrusters that produce energy and/or trigger proximity explosions at greater than normal range are both far more powerful benefits than any amount of reverse thrust ever could be.

That doesn't mean they shouldn't have reverse thrust. They should have reverse thrust.

**Reply by Zitchas** [reply]
That is just "Hunan tech but better." Boring. Much more interesting to give them zero, or at least very low inertia vs thrust, such that they come to a complete stop almost as soon as they stop thrusting. It isn't reverse thrust, they have no need for that. But it is the capacity to enter optimum range, and come to a stop right there without spinning around. That would be both unique, interesting, and fitting. And very powerful and effective.

**Reply by Arachi-Lover** [reply]
> That is just "Hunan tech but better." Boring. Much more interesting to give them zero, or at least very low inertia vs thrust, such that they come to a complete stop almost as soon as they stop thrusting.

Okay, so as I said give that to them on top of the reverse thrust, and they have even more tools to mark their advancements.

> It isn't reverse thrust, they have no need for that.

So are we just going to ignore the fact I brought up that reverse thrust objectively is better for getting back to the target in case the ship overshoots than simple turning? With your proposal to have them stop immediately afterwards that'd be even more helpful in doing that. Yes, even with turrets it's helpful. Especially with turrets, actually, as reverse thrust is less useful for gun-based ships as that type of kiting requires them to also have the turning to keep with the direction they're firing at. Turrets don't need that, so reverse thrusting is a benefit by itself, and will not be diminished by a potential lack of another attribute such as turning.

> But it is the capacity to enter optimum range, and come to a stop right there without spinning around.

Reverse thrust would also do that, and a combo of both would mean they can start reversing immediately, making it much more effective at backtracking their path as they don't waste time slowing down their forward thrust only to then start thrusting back. So, both.

> That would be both unique, interesting, and fitting. And very powerful and effective.

And them having reverse thrust as well wouldn't diminish any of that, it'd only add to their tools and uniqueness of their engines, as the reverse paired with the inertia would result in aforementioned positives.

**Reply by Zitchas** [reply]
You and I have very different definitions of "uniqueness", I think. Adding reverse thrust is never going to increase how unique they are. That ship has sailed, a very long time ago. The only thing reverse thrust adds at this point is making these engines even more "do everything" than they might already be.

**Reply by lumbar527** [reply]
I think I agree with Zitchas that reverse thrust isn't very unique. The Remnant already have that as a primary engine factor, and you can get it easily enough with specialized outfits.

What if the Quarg had a little bit of reverse thrust and another special attribute? (I especially like the "inertialess" idea)
That way they would have the usefulness of reverse thrust while also being special a different way.

**Reply by Arachi-Lover** [reply]
Does adding reverse thrust in any way reduce the uniqueness of the other proposed attributes?
If so, why and how?

It seems to me this opposition is purely stemming from a desire to have only the Remnant engines have built-in reverse thrust, and thus looking for replacement stats to use instead in the Quarg engines, despite it being the case that for years now nearly every single discussion on how we could improve on the Quarg engines has revolved around the thought of making their engines "omnidirectional" which fits right in with reverse thrust.

And, if we're so worried that having reverse thrust would be making them "do everything" engines, then where's the concern with that for Remnant engines having slowing resistance? Or are we removing that from Remnant engines so as to not have them be "doing everything"?

I don't think any of the proposed other attributes are bad to include on Quarg engines, but I do not see why it needs to come at the expense of the proposed reverse thrust.

**Reply by lumbar527** [reply]
I just changed my mind: reverse thrust equal to thrust would be great along with no inertia. That sounds like something gravitational engines might be able to have; inertialess omnidirectional engines.

**Reply by Saugia** [reply]
Personally, I also don't see why giving the Quarg reverse thrust in their engines is a bad thing, and in fact, it would make sense that the Quarg of all species in the game have engines that do both. We can talk about adding other special attributes during a later time, but I don't think that's reason to take away from the Quarg having integrated reverse thrust. 

It is true that the Remnant also do this, and they have good engines for it, but they aren't the only ones that do it and there's no reason for them to be the only ones to do it. The Ka'het do it, and actually, one of the Gegno's larger thrust systems also does it. If the Gegno can do it, there's no reason why the Quarg can't. If the idea is that it's "boring", that's not a mechanical reason not to do it, it is a subjective opinion on the matter.

**Reply by Azure3141** [reply]
I would add that if lateral thrust is ever added, it would make sense for the Quarg to have that in equal amounts to their axial thrust as well. It also might make sense to combine their turning engines with their thrusting engines, to provide the sense that their engines aren't really thrusters but are instead full six axis gravity drives.

**Reply by EjoThims** [reply]
Second combining it all into one. I think we should go so far as to put in a comment for intent of matching lateral thrust when it's included.

**Reply by Quantumshark** [reply]
Integrated 100% reverse thrust would be reasonable balance-wise, but I agree with Zitchas' comments that it would be preferable to give Quarg engines something else interesting _instead_.
I don't personally agree with the idea that they should just have everything there is because they're T3 and thus more advanced.

As a more radical proposal, which I feel is thematically fitting but probably unbalanceable, consider this:
We know the Quarg sometimes carry extremely large amounts of material around. They must do, because they dismantle entire systems' worth of matter and transport it to their rings (the current numbers, given that we know they take around 10,000 years to finish a ringworld, suggest that they're moving around 10^15 tons of material every day on average).
One notable feature of gravity as a force is that the acceleration of a falling body due to gravity is not affected by the mass of that body.

So, linking these two points together, how about giving Quarg thrusters a new "fixed acceleration" mechanic - regardless of the mass of the ship they're on, that ship always has a set amount of acceleration, fixed by that stat on the thrusters.
The thrusters would also have a thrust stat, but it'd only affect top speed, and for acceleration it would be overridden by the fixed acceleration.

**Reply by Arachi-Lover** [reply]
I don't believe anyone has proposed making Quarg engines have everything there is.
The suggestions have been to add reverse thrust, the inertia thing, energy production, and something to do with jamming, and now your suggestion of the fixed acceleration.

The way I see it, reverse thrust is the most useful one for Quarg ships from a tactical standpoint, as it allows for much easier and faster repositioning.
Inertia, jamming, energy or acceleration would be extras on top of that. Any of those could be picked, personally I think the inertia idea would fit better with the whole omnidirectional movement concept.

These are Quarg engines, they *should* be unique and have lots to them. That doesn't mean having everything, or even all of the ideas proposed there. Say, two of the attributes proposed are used.
So, reverse thrust, which would be the most useful and most fitting for engines, and some other more exotic attribute.

This wouldn't make them any more out there than the existing Remnant engines, which have all their own engine stuff, reverse thrust, and slowing resistance.
Quarg engines would have all their own engine stuff, reverse thrust, and one of the more exotic attributes.

**Reply by Hecter94** [reply]
> So, linking these two points together, how about giving Quarg thrusters a new "fixed acceleration" mechanic - regardless of the mass of the ship they're on, that ship always has a set amount of acceleration, fixed by that stat on the thrusters.

This feels more like something I'd expect to see on Efreti engines. Since they both use gravitational engines *and* have large ships, whereas Quarg are going to be focused more on having smaller ships on average and no particularly large ones, so they'd get little benefit from this fixed acceleration. 

I personally feel Omnidirectional engines are the best route to take for them, with equal forward and reverse, and lateral when/if that happens.

**Reply by RisingLeaf** [reply]
Another thing that could be unique to those engines, which is not possible with current mechanics but not too difficult to implement, would be for them to either pull enemies close or push them away while using them, as they are manipulating gravity.
Depending on what fits their weaponry better it could give them a strong advantage in battlefield.

**Reply by lumbar527** [reply]
I think it makes more sense for them to be using omnidirectional engines, rather than using their engines directly for combat. Maybe.

**Reply by Saugia** [reply]
On the other side of things, I'm not really convinced the Quarg need some brand new neat mechanic just because they're Tier 3 and getting a revamp either. The less exclusive mechanics in the game, the better. If we give the Quarg something nobody else has, that would be at odds with several other species like their relation with the Drak and Pug. I don't think there's anything wrong with using what we have instead of trying to make them "cooler". 

That's not to say I would be against some type of gravitational advantage in Quarg design, but that doesn't need to be, and likely shouldn't be, an effect of the engines. Having reverse thrust does not give any sort of detriment besides the opinion of that it's boring and not unique enough to the Quarg, which as I said is subjective reasoning, not a mechanical or balance reasoning. There's nothing wrong with adding something that currently exists in the game to a faction currently underwhelming in some areas.

I'm not really sure if we're adding lateral thrust to the game as it's been said we won't be doing strafing mechanics, but that's a discussion for another topic. Remember, this revamp is not necessarily an attempt to or invitation to add new mechanics just for the Quarg. If that's something desired, it can easily be discussed after. A majority of this revamp is visuals, logistics, and outfits, not new mechanics or code, i.e., like with creating new factions, they shouldn't rely on the creation of a new mechanic.

**Reply by DarcyManoel** [reply]
I have an idea on the integrated engines. Turning should remain as a second set, but require the thrusters to be installed first.

Having this set up would restrict Quarg engines from being as flexible for mixed faction builds. Sense being that a second gravity engine would be required to actually turn rotate the ship in an opposing direction as the first is.

Turning engines would include identical thrust as the first set, but enable very powerful turning force. Naturally, adding a second turning engine would increase the thrust of the build by 50% but turning by 100%, a third would increase the thrust of the build by 33% and turn by 50%.

**Reply by palant** [reply]
> Engines are in a reasonable state already apart from the aformentioned energy issue, though they don't exactly scream "T3". The suggestion to give them reverse thrust equal to their normal thrust seems reasonable from the balance side, though integrated reverse thrust is already a feature of Remnant engines and is planned for the Xapleaux, so perhaps it'd be better to give Quarg engines something else instead.

I think that a civilization which discovered folding space to jump from one place to another should be able to make their ships turn around without changing their speed. From full speed ahead to full speed into opposite direction in two seconds, sort of like a slingshot around an invisible singularity that impacts only their motion. If a T3 civilization bends the laws of physics, it should do so in a very visible and obvious way.

Would make for some interesting interactions if equipped together with other species’ engines. I can see two options:

1. Installing a conventional steering engine with a Quarg engine disrupts the latter and is damaging to the ship. As such, it should be disallowed by preflight checks – or maybe the players can be allowed to discover it the hard way.
2. A conventional steering engine and a Quarg engine *can* work together, the former rotating the ship around its axis while the latter turns it around. This interaction would make the ship very hard to steer, making such combinations impractical.

**Comment by RisingLeaf** [comment]
Regarding projectile sprites, the ranseur and the skylance look nearly exactly the same, should it not be easier for the player to see what they get attacked with?

**Reply by Saugia** [reply]
I'm looking into working on the visuals of weapons as well as sounds. I have some ideas for changing how they look and operate. Will post updates.

**Comment by RisingLeaf** [comment]
Two things:

1. With an expansion of the Quarg, should come a deepening of the pug lore. Of course that's not the core point of this rework but the Quarg and the Pug are strongly related lore wise and if the Quarg lore gets expanded this should also be done under the consideration of the Pug Lore.

2. I think it would make sense to create another project room on Discord? A discussion is good for talking about bigger things but a discord chat room offers a more casual way to talk about things.

**Reply by Saugia** [reply]
I'll bring up the channel on discord. As for the Pug, I'm not sure. One thing that *will* have to likely be done is rebalancing Pug ships and outfits around the new Quarg, but as for a revamp, I'm not really sure Pug lore needs to be deepened. 

We are deepening Quarg lore, yes, but it's majorly visuals, and the Pug technically always had this with their tier-difference ships as well as their splinter groups, involvement in different parts of the campaign, etc., whereas the Quarg were never much beyond their Ringworlds and Wardragons. 

At the moment the Pug have more "active lore", if you will, and this revamp will give the Quarg more "passive lore". I think any deepening of Pug lore should just be considered the same as it's always been, as well as how Quarg has been, by interactions in future content or something, not just because the Quarg are getting something.  

Remember, the Quarg have been placeholders for years, the Pug have not. Although both species are mysterious, it makes sense to keep the Pug as more of an unknown being an extra-galactic species, not a species that has been home to the galaxy for a long time.

**Reply by RisingLeaf** [reply]
I see your point and after reading this I agree.

**Reply by mathwhiz1212** [reply]
@Saugia Isn't "home to the Galaxy for a long time" somewhat relative considering the ES storyline at least implies that the Quarg and Pug have been fighting in or around the human galaxy since before humans were around?

Though the Pug have never really had a "home" in the human galaxy if that's what you mean.

I agree with your reasonings about lore, I'm just curious about this particular point.

**Reply by Saugia** [reply]
Yes, that's what I mean - the Pug are Extragalactic, and the Quarg are not. It makes sense the Pug remain a mysterious factor.

**Reply by mathwhiz1212** [reply]
That makes sense.

**Comment by OcelotWalrus** [comment]
Hey,

Sorry to bother you guys, but, is there any way to get the blender/3D models of any ship that looks like this one?
![quarg??](https://github.com/endless-sky/endless-sky/assets/87318892/40da9cb1-1109-4d62-b819-f22096fae0fe)

**Reply by EjoThims** [reply]
I think @Arachi-Lover probably is most likely to have or know where to get them all.

I think maybe? Brick as well, but I'm not nearly positive enough on his name here to tag the random suggestion that comes up.

**Reply by OcelotWalrus** [reply]
Thanks!

**Reply by Arachi-Lover** [reply]
Assets can be found on the old PR that was also a try at a Quarg expansion:

https://github.com/endless-sky/endless-sky/pull/3665

https://drive.google.com/drive/folders/1qCnJMyfmY2Za6EFqBz2k4TqLFwHofdNS

**Reply by OcelotWalrus** [reply]
Thanks!

**Comment by SultanRTG** [comment]
What do you guys use to make the models for Endless Sky?

**Reply by Saugia** [reply]
Blender and GIMP, generally.

**Comment by TheGiraffe3** [comment]
The Quarg Revamp was done in #11102, so I'm closing this.
